moveit_ros/moveit_servo/CMakeLists.txt

```diff
@@ -7,14 +7,18 @@ endif()
 set(CMAKE_CXX_STANDARD_REQUIRED ON)
 set(CMAKE_CXX_EXTENSIONS OFF)
 
-set(LIBRARY_NAME moveit_servo_cpp_api)
+set(SERVO_LIB_NAME moveit_servo_cpp_api)
 
 if(NOT CMAKE_CONFIGURATION_TYPES AND NOT CMAKE_BUILD_TYPE)
   set(CMAKE_BUILD_TYPE Release)
 endif()
 
-find_package(Boost REQUIRED)
+# For library compilation issue. See
+# https://stackoverflow.com/questions/25902519/building-error-cmake-can-not-be-used-when-making-a-shared-object-recompile-wi
+SET(CMAKE_CXX_FLAGS "${CMAKE_CXX_FLAGS} -fPIC")
```

Comment:
You don't need this if we build libraries as shared objects.
If you add it, you also remove a part of why people even want static libraries to begin with...